src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -49,32 +50,22 @@
                 <FluentAnchor slot="end" Appearance="Appearance.Lightweight" Href="@DashboardUrls.StructuredLogsUrl(traceId: trace.TraceId)">@ControlStringsLoc[nameof(ControlsStrings.ViewLogsLink)]</FluentAnchor>
             </ToolbarSection>
             <MainSection>
-                <SummaryDetailsView ShowDetails="SelectedSpan is not null"
-                                    OnDismiss="@(() => ClearSelectedSpanAsync(causedByUserAction: true))"
-                                    ViewKey="TraceDetail"
-                                    SelectedValue="@SelectedSpan">
+                <SummaryDetailsView
+                    ShowDetails="SelectedSpan is not null"
+                    OnDismiss="@(() => ClearSelectedSpanAsync(causedByUserAction: true))"
+                    ViewKey="TraceDetail"
+                    SelectedValue="@SelectedSpan">
                     <DetailsTitleTemplate>
-                        @{
-                            var shortedSpanId = OtlpHelpers.ToShortenedId(context!.Span.SpanId);
-                        }
+                        @{ var shortedSpanId = OtlpHelpers.ToShortenedId(context!.Span.SpanId); }
                         <div class="pane-details-title" title="@($"{context!.Title} ({shortedSpanId})")">
                             @context!.Title
                             <span class="pane-details-subtext">@shortedSpanId</span>
                         </div>
                     </DetailsTitleTemplate>
                     <Summary>
-                        <FluentDataGrid Virtualize="true"
-                                        GenerateHeader="GenerateHeaderOption.Sticky"
-                                        Class="trace-view-grid enable-row-click"
-                                        ResizableColumns="true"
-                                        ItemsProvider="@GetData"
-                                        TGridItem="SpanWaterfallViewModel"
-                                        RowClass="@GetRowClass"
-                                        GridTemplateColumns="4fr 12fr 85px"
-                                        ShowHover="true"
-                                        OnRowClick="@(r => r.ExecuteOnDefault(d => OnShowPropertiesAsync(d, buttonId: null)))">
-                            <TemplateColumn Title="Name">
-                                <div class="col-long-content" title="@context.GetTooltip(_applications)">
+                        <FluentDataGrid Virtualize="true" GenerateHeader="GenerateHeaderOption.Sticky" Class="trace-view-grid" ResizableColumns="true" ItemsProvider="@GetData" TGridItem="SpanWaterfallViewModel" RowClass="@GetRowClass" GridTemplateColumns="@_manager.GetGridTemplateColumns()" ShowHover="true">
```

Comment:
Use the previous format of attribltes on multiple lines. Also, row click is still removed here.